tests/features/event.feature

```diff
@@ -0,0 +1,260 @@
+    And I should see "Featured media legend"
+    And I should see "Full text"
+    And I press "Description"
+
+    When I press "Report"
+    Then I should see "Summary for report"
+    And I should see "Report title"
+    And I should see "Report text"
+    And I press "Report"
+
+    When I press "Registration"
+    Then I should see "Registration URL"
+    And I should see "Registration status"
+    And I should see "Registration start date"
+    And I should see "Registration end date"
+    And I should see "Entrance fee"
+    And I should see "Registration capacity"
+
+  @cleanup:media @javascript
+  Scenario: Creation of a Event content through the UI.
+    Given I am logged in as a user with the "create oe_event content, access content, edit own oe_event content, view published skos concept entities, create av_portal_photo media" permission
+    # Create a "Media AV portal photo".
+    And I visit "the AV Portal photo creation page"
+    And I fill in "Media AV Portal Photo" with "https://audiovisual.ec.europa.eu/en/photo/P-038924~2F00-15"
+    And I press "Save"
+    # Create a "Event" content.
+    And I visit "the Event creation page"
+    And I fill in "Title" with "My Event item"
+    And I fill in "Start date" with the date "02/21/2019"
+    And I fill in "Start date" with the time "02:21:00AM"
+    And I fill in "End date" with the date "02/21/2019"
+    And I fill in "End date" with the time "02:21:00PM"
+    When I select "As planned" from "Status"
+    Then I should have the following options for the "Status" select:
+      | - Select a value - |
+      | As planned         |
+      | Cancelled          |
+      | Rescheduled        |
+      | Postponed          |
+    When I select "Info days" from "Type"
+    Then I should have the following options for the "Type" select:
+      | Training and workshops            |
+      | Info days                         |
+      | Competitions and award ceremonies |
+      | Conferences and summits           |
+      | Public debates                    |
+      | Partner meetings                  |
+      | Political meetings                |
+      | Exhibitions                       |
+    And I fill in "Subject" with "financing"
+    And I fill in "URL" with "http://ec.europa.eu"
+    And I fill in "Link text" with "Website"
+    # Online field group.
+    When I press "Online"
+    And I select "Facebook" from "Online type"
+    Then I should have the following options for the "Online type" select:
+      | - None -   |
+      | Facebook   |
+      | Livestream |
+    And I fill in "Online time start" with the date "02/22/2019"
+    And I fill in "Online time start" with the time "02:22:00AM"
+    And I fill in "Online time end" with the date "02/22/2019"
+    And I fill in "Online time end" with the time "02:22:00PM"
+    And I fill in "Online description" with "Online description text"
+    And I fill in "URL" with "http://ec.europa.eu/2" in the "online link"
```

Comment:
`And I fill in "URL" with "http://ec.europa.eu/2" in the "Online link" region`? Same for below